base_classes/NXdetector.nxdl.xml

```diff
@@ -89,7 +89,28 @@
   </field>
 
   <field name="data" type="NX_NUMBER" units="NX_ANY">
-    <doc>Data values from the detector.</doc>
+    <doc>
+      Data values from the detector. The base dimensionality of this data is detector dependent.
+      If it is only a single value, it is just one value. If it is a linear detector, it is one dimensional
+      with the  number of pixels in the linear detector as dimension, [npixel].  
+      
+      If it is an image, then it is 2D, [xdim, ydim]. If the detector is three dimensional, then the 
```

Comment:
Should we use fast/slow language instead of xdim, ydim?

Reply:
xdim and ydim sound more familiar to me; fast and slow is another abstraction level which does help when talking about storage order but not here. My 2c worth.

Reply:
It's not really xdim/ydim as names, it's the order.  In many datasets the order is [ydim,xdim], assuming a single panel detector aligned to the lab axes.  If instead it were [slow,fast] it would be unambiguous.  I think x/y should be concepts for NXtransformations, and fast/slow should be concepts for data.